diesel_derives/src/lib.rs

```diff
@@ -77,10 +75,10 @@ use diagnostic_shim::*;
 /// the derive should threat `None` values as `NULL`. By default
 /// `Option::<T>::None` is just skipped. To insert a `NULL` using default
 /// behavior use `Option::<Option<T>>::Some(None)`
-/// * `#[table_name = "some_table"]`, specifies the table for which the
-/// current type is a changeset. Requires that `some_table` is in scope.
+/// * `#[table_name = "path::to::table"]`, specifies a path to the table for which the
+/// current type is a changeset.
```

Comment:
Should I add "Requires that `path` is in scope." here? It seems kinda obvious to me that the path is relative to the current scope.

Reply:
Stating the obvious here cannot hurt, therefore I believe adding something like: "The path is relative to the current module" is a good idea.

Reply:
done